src/overlays/actors/ovl_Demo_Gt/z_demo_gt.c

```diff
@@ -364,23 +358,22 @@ void func_8097E744(DemoGt* this, GlobalContext* globalCtx, u32 actionIdx) {
     }
 }
 
-#ifdef NON_MATCHING
 void func_8097E824(DemoGt* this, s32 arg1) {
     s16 phi_a1;
     s16 phi_a2;
     s16 phi_a3;
+    Vec3f* pos = &this->dyna.actor.posRot.pos;
+    Vec3s* unk16C = &this->unk_16C;
+    s32 pad;
 
```

Comment:
remove any extra line breaks in declaration list, i must have missed them when i was cleaning the file up